python_examples/basic_simulation/simple_parameters.json

```diff
@@ -309,6 +309,40 @@
         0.001105933, 0.00091683, 0.000759816, 0.000629496,
         0.000521372, 0.000431695, 0.000357344, 0.000295719,
         0.000244659],
+    "place_params": {
```

Comment:
We have avoided putting places into the basic simulation example, as we wanted it to be the simplest example possible. Could we maybe put these parameters into our Gibraltar simulation instead (or the age stratified one) as these are meant to be more complete, and include 'advanced' features such as carehomes?

Reply:
I agree with Kit. I think it would be better to add it to the Gibraltar simulation (or create a new simulation)
simple_parameters is also the only example that I've adapted the json file into an order based on parameter importance but that's on a branch of the fork